ajax/libs/DragDrop/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "DragDrop",
+  "filename": "drag-drop.min.js",
+  "version": "0.3.0",
+  "description": "A JavaScript micro-framework for adding drag-and-drop functionality to elements for advanced UI development",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kbjr/DragDrop.git"
+  },
+  "author": "James Brumond",
+  "homepage": "https://kbjr.github.io/DragDrop/index.html",
+  "keywords": [
+    "drag",
+    "drop"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/kbjr/DragDrop.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "drag-drop.*"
```

Comment:
Let's modify this to `drag-drop*` to prevent possible postfix.
We can notice there is a file `jquery.dragdrop.js` in master branch. Let's add another rule `jquery.dragdrop*` for this file to prevent it appear in the future version.
Don't forget rebase your branch.

Reply:
Should I add jquery.dragdrop.js to version file
or just add another rule jquery.dragdrop* in package.json. Thanks.

Reply:
@finch185277 Just add the rule. Because `jquery.dragdrop.js` is only on master branch, which means it might appear **in the future version**. For current v0.3.0, only `drag-drop.js` & `drag-drop.min.js` are there.